Description: Fixes
 .
 dom/media/webrtc/transport/third_party/nrappkit/src/util/util.cpp:110:8: error: redefinition of a 'extern inline' function 'strlcat' is not supported in C++
   110 | size_t strlcat(char *dst, const char *src, size_t siz)
       |        ^
 /usr/include/x86_64-linux-gnu/bits/string_fortified.h:167:8: note: previous definition is here
   167 | __NTH (strlcat (char *__restrict __dest, const char *__restrict __src,
       |        ^
 .
Author: Nathan Pratta Teodosio <nteodosio@ubuntu.com>

--- a/dom/media/webrtc/transport/third_party/nrappkit/src/util/util.cpp
+++ b/dom/media/webrtc/transport/third_party/nrappkit/src/util/util.cpp
@@ -71,5 +71,5 @@
   }
 
 
-#if defined(LINUX) || defined(WIN32)
+#if defined(WIN32)
 /*-
  * Copyright (c) 1998 Todd C. Miller <Todd.Miller@courtesan.com>
  * All rights reserved.

